src/components/account/LoginCard.tsx

```diff
@@ -0,0 +1,87 @@
+import { Button, Checkbox, Form, Input } from 'antd';
+import React, { CSSProperties, ReactElement } from 'react';
+
+const styles: { [identifier: string]: CSSProperties } = {
+  button: {
+    float: 'left',
+  },
+  inputWrapper: {
+    marginBottom: 15,
+  },
+};
+
+const layout = {
+  labelCol: { span: 4 },
+  wrapperCol: { span: 8 },
+};
+const tailLayout = {
+  wrapperCol: { offset: 4, span: 8 },
+};
+interface Props {
+  onClickRegister: () => void;
+}
+
+export default function LoginCard(props: Props): ReactElement {
+  const onFinish = () => {
+    alert('Finish');
+    //console.log('Success:', values);
+  };
+
+  const onFinishFailed = () => {
+    //console.log('Failed:', errorInfo);
+  };
```

Comment:
Could we link the tickets as todo comments here? If the tickets don't exist in the `User Authentication` epic in JIRA, would you mind creating them? That way we can keep track of what's left in the FE.

Reply:
Sure!